src/panels/SkyBrowserPanel/ImageList/ImageList.tsx

```diff
@@ -36,7 +33,7 @@ export const ImageList = memo(function ImageList({
   const columns = Math.max(Math.min(Math.floor(width / cardWidth), maxColumns), 1);
 
   if (imageList.length === 0) {
-    return noImagesDisplay;
+    return noImagesDisplay ?? <Text>No images found, try another search!</Text>;
```

Comment:
This does not seem to be displayed?
![image](https://github.com/user-attachments/assets/1f1392ad-0810-4128-8ad5-2d22798f0515)